.github/workflows/build-and-push-docker-image.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+name: 🏗️
+
+on:   # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches: ["release/*", "unstable/*"]
+    tags: ["*"]
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - uses: actions/checkout@v4
+    - name: Build Docker image
+      run: |
+        IMAGE="ghcr.io/$GITHUB_REPOSITORY:${GITHUB_REF_NAME/'/'/'-'}"
+        echo "IMAGE=$IMAGE" >>"$GITHUB_ENV"
+        docker build . \
+          --build-arg BUILDKIT_INLINE_CACHE=1 \
+          --cache-from $IMAGE \
+          --tag $IMAGE
+    - name: Push Docker image to GHCR
+      if: github.event_name != 'pull_request'
+      run: |
+        echo ${{ secrets.GITHUB_TOKEN }} |
```

Comment:
Doesn't this need a `\` at the end? Can this be split into two steps?

Reply:
Separating the `docker login` and `docker push` steps is a good idea. I've pushed that change (bb0315f).

There's no `\` needed in the `docker login`. The `|` with next line indented implies that the statement continues on the next line. This syntax also helps stay under the 80 character `yamllint` line length limit.

Reply:
Thanks! I'll mark this thread as resolved.